Assets/__Scripts/BeatSaberSong.cs

```diff
@@ -220,6 +221,30 @@ public JSONNode ToJSONNode()
         }
     }
 
+    public DateTime LastWriteTime;
+    private bool _isFavourite;
+    public bool IsFavourite
+    {
+        get => _isFavourite;
+        set
+        {
+            var path = Path.Combine(directory, ".favourite");
```

Comment:
oi this is quite bri'ish of you, innit?

Reply:
Time to rename to `.fav` or `.star` to avoid this debate 👀

Reply:
brb gonna get me some tea n' crumpets

~~i honestly dont care, just like poking fun~~